common/lib/xblock/__init__.py

```diff
@@ -0,0 +1,10 @@
+'''
```

Comment:
Also, can we name it something else, like `builtin_xblocks` or something? Using `pkgutil.extend_path` is perfectly valid, but it's not known, and it may confuse people if we're sharing the xblock namespace.